cpp/pagerank_module/algorithm/pagerank.cpp

```diff
@@ -0,0 +1,147 @@
+#include <algorithm>
+#include <numeric>
+#include <queue>
+
+#include "pagerank.hpp"
+
+namespace pagerank_alg {
+
+PageRankGraph::PageRankGraph(const std::uint64_t number_of_nodes, const std::uint64_t number_of_edges,
+                             const std::vector<EdgePair> &edges)
+    : node_count_(number_of_nodes),
+      edge_count_(number_of_edges),
+      out_degree_(std::vector<std::uint64_t>(number_of_nodes)) {
```

Comment:
this is just a call to constructor of those fields so you can do
```
out_degree_(number_of_nodes)
```
directly. Just remember not to use `{}` as you would initialize vector with one element of value `number_of_nodes`. :smile: